sema4ai/vscode-client/src/robo/agentPackage.ts

```diff
@@ -154,9 +155,8 @@ export const packAgentPackage = async (targetDir: string): Promise<{ zipPath: st
         const errorMsg = `Failed to package the agent at: ${targetDir}`;
         logError(errorMsg, error, "ERR_PACK_ACTION_PACKAGE");
 
-        const detailedErrorMsg = `${errorMsg}. Please check the 'OUTPUT > Sema4.ai' for more details.`;
-        OUTPUT_CHANNEL.appendLine(detailedErrorMsg);
-        window.showErrorMessage(detailedErrorMsg);
+        OUTPUT_CHANNEL.appendLine(errorMsg);
```

Comment:
If logError(errorMsg) is used, the OUTPUT_CHANNEL.appendLine(errorMsg); is not needed (as logError will already add that to the OUTPUT_CHANNEL).